assets/src/components/v2/cr_departures.tsx

```diff
@@ -21,51 +24,73 @@ interface CRDeparturesProps {
   departures: Departure[];
   destination: string;
   time_to_destination: string;
-  show_via_headsigns_message: boolean;
+  overnight_asset_url: string;
 }
 
 const DeparturesTable: React.ComponentType<any> = (props) => {
   const { departures } = props;
 
-  let [headsignPageOne, setHeadsignPageOne] = useState(true);
+  const getArrowOrTbd = (arrow: string) => {
+    if (arrow) {
+      return (
+        <div className="arrow-image">
+          <Arrow
+            direction={arrow as Direction}
+            className="departure__arrow-image"
+          />
+        </div>
+      );
+    }
 
-  useInterval(() => {
-    setHeadsignPageOne(!headsignPageOne);
-  }, 4000);
+    return <div className="track-tbd">TBD</div>;
+  };
+
+  const getStationServiceList = (stationServiceList: StationService[]) => {
+    return stationServiceList.map((station: StationService) => {
+      return (
+        <div className="stops-at-text" key={station.name}>
+          <div className="via-service-icon">
+            <img
+              src={imagePath(
+                station.service ? "cr-service.svg" : "cr-no-service.svg"
+              )}
+            />
+          </div>
+          <div className="via-stop-name">{station.name}</div>
+        </div>
+      );
+    });
+  };
 
   return (
     <table>
       <tbody>
         <tr>
-          <td className="arrow"></td>
+          <td className="track">
+            <div className="table-header__english">Track</div>
+            <div className="table-header__spanish">Pista</div>
+          </td>
           <td className="headsign">
             <div className="table-header__english">Upcoming departures</div>
             <div className="table-header__spanish">Próximas salidas</div>
           </td>
           <td className="arrival"></td>
-          <td className="track">
-            <div className="table-header__english">Track</div>
-            <div className="table-header__spanish">Pista</div>
-          </td>
         </tr>
-        {departures.slice(0, 3).map((departure) => {
+        {departures.slice(0, 3).map((departure: Departure) => {
```

Comment:
Is this a merge conflict? I moved the filtering to the backend (line 34 in cr_departures.ex), but maybe it doesn't hurt to have the double filtering.

Reply:
The slicing is the part I was questioning -- the I previously removed that slice and moved it to the backend, but the slice is back